src/main/java/net/sourceforge/plantuml/activitydiagram3/command/CommandActivityLegacy1.java

```diff
@@ -59,14 +64,32 @@ static IRegex getRegexConcat() {
 		return RegexConcat.build(CommandActivityLegacy1.class.getName(), RegexLeaf.start(), //
 				new RegexLeaf("[-*]"), //
 				RegexLeaf.spaceZeroOrOne(), //
+				StereotypePattern.optional("STEREO1"), //
 				new RegexLeaf(1, "LABEL", "(.*)"), //
```

Comment:
Hi @arnaudroques 

It seems the `Label` regex is eating everything with `(.*)`!
How can we make it leave some for the others?

```puml
@startuml
- <<icon>><:bug:>
- <:bug:> <<icon>> [[plantuml.com]]
@enduml
```
>[![](https://img.plantuml.biz/plantuml/svg/SoWkIImgAStDuUNILR2noKpEpxEpix5AAastiWCBGPXmAONew8ASn3oG3hta_DpOM5wkjFpyajGY1TMorABYpFmy3KrUBeVKl1IW2W00)](https://editor.plantuml.com/uml/SoWkIImgAStDuUNILR2noKpEpxEpix5AAastiWCBGPXmAONew8ASn3oG3hta_DpOM5wkjFpyajGY1TMorABYpFmy3KrUBeVKl1IW2W00)

Reply:
Fixed by:
- #2456